salt/top.sls

```diff
@@ -9,6 +9,7 @@ base:
     - proxy
     - rebootmgr
     - transactional-update
+    - kube-apiserver-proxy
```

Comment:
Do we need this new top-level dir? It looks to me the code in  `salt/kube-apiserver-proxy/init.sls` (it is just a certificca generation) could be just moved to `salt/haproxy/init.sls`

Reply:
Yes, I'll move it.